app/lib/actions/katello/repository/clone_ostree_content.rb

```diff
@@ -0,0 +1,15 @@
+module Actions
+  module Katello
+    module Repository
+      class CloneDockerContent < Actions::Base
+        def plan(source_repo, target_repo)
+          sequence do
+            # TODO: clone ostree branches?
```

Comment:
Cloning them is probably good idea, because we allow user to do repository info --id=<cloned-repo> and they'll want branch info there